newIDE/app/src/InstancesEditor/HighlightedInstance.js

```diff
@@ -47,5 +56,37 @@ export default class InstancesSelection {
       highlightRectangle.height
     );
     this.highlightRectangle.endFill();
+
+    const tooltipInfo =
+      'Object: ' +
```

Comment:
Let's remove "Object: " - it should be fairly obvious that this is the object name no?
Also will save some space on screen